ProjFS.Mac/PrjFS.xcodeproj/project.pbxproj

```diff
@@ -166,7 +171,7 @@
 		4391F89021E42AC40008103C /* kernel-header-wrappers */ = {isa = PBXFileReference; lastKnownFileType = folder; path = "kernel-header-wrappers"; sourceTree = "<group>"; };
 		4391F89121E42AC40008103C /* KextLog.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = KextLog.cpp; sourceTree = "<group>"; };
 		4391F89221E42AC40008103C /* KextLog.hpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.h; path = KextLog.hpp; sourceTree = "<group>"; };
-		4391F89321E42AC40008103C /* KauthHandler.cpp */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.cpp.cpp; path = KauthHandler.cpp; sourceTree = "<group>"; };
+		4391F89321E42AC40008103C /* KauthHandler.cpp */ = {isa = PBXFileReference; fileEncoding = 4; indentWidth = 4; lastKnownFileType = sourcecode.cpp.cpp; path = KauthHandler.cpp; sourceTree = "<group>"; usesTabs = 0; };
```

Comment:
That's probably because I seem to run into an Xcode indentation bug a lot and actively have to reset the indentation settings whenever I change git branch. I'll remove this change.